src/main/java/gregtech/common/blocks/VariantBlock.java

```diff
@@ -53,6 +60,16 @@ protected BlockStateContainer createBlockState() {
         return new BlockStateContainer(this, VARIANT);
     }
 
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, ITooltipFlag advanced) {
+        String unlocalizedVariantTooltip = getUnlocalizedName() + ".tooltip";
```

Comment:
can't see any reason why there should be both

Reply:
That is same as "tier less" for electrical machines. Because getUnlocalizedName() doesn't have last part of name but stack.getUnlocalizedName() has it. So with this you can both have tool tips on "category" and specific item. However it should be at least properly commented.